### PR TITLE
update concourse to 7.3.x

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- update concourse to 7.3.x

## security considerations
staying up-to-date means we're less likely to run with know vulns

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -161,7 +160,6 @@ jobs:
       - concourse-config/operations/set-garbage-collection.yml
       - concourse-config/operations/base-resource-defaults.yml
       - concourse-config/operations/bosh-dns-aliases.yml
-      - concourse-config/operations/use-bionic.yml
```
Comment (ccostino) [inline ci/pipeline.yml:164]: Just curious what this was and that we don't need it anymore?  Was it something in place while we were transitioning from `xenial`?

Comment (bengerman13) [reply ci/pipeline.yml:164]: before 7.3.0, the concourse manifest specified xenial stemcells, and this opsfile overrode that. In 7.3.0 they updated the manifest to specify bionic, so our bionic opsfile now breaks the deployment

Comment (ccostino) [reply ci/pipeline.yml:164]: That's what I figured, but wanted to be sure.  Thanks!

**Review by ccostino — APPROVED** [review]
Thanks, @bengerman13!

**Pull request merged**